src/docs/truffle-dashboard/quickstart.md

```diff
@@ -21,28 +30,18 @@ DashboardProvider RPC endpoint running at http://localhost:24012/rpc
 
 By default, the command above starts a dashboard at `http://localhost:24012` and opens the dashboard in a new tab in your default browser. The Dashboard then prompts you to connect your wallet and confirm that you're connected to the right network. **You should double check your connected network at this point, since switching to a different network during a deployment can have unintended consequences.**
 
-![Connect your wallet to the Truffle Dashboard](/img/docs/truffle/using-the-truffle-dashboard/truffle-dashboard-connect.png)
-
-![Confirm your connected network](/img/docs/truffle/using-the-truffle-dashboard/truffle-dashboard-confirm.png)
+![Connect your wallet to the Truffle Dashboard](/img/docs/truffle-dashboard/quickstart/connect-wallet.png)
 
-The port and host can be configured through command line options, or by configuring them inside your `truffle-config.js`.
-
-```js
-module.exports = {
-  // ... rest of truffle config
+![Confirm your connected network](/img/docs/truffle-dashboard/quickstart/confirm-network.png)
 
-  dashboard: {
-    port: 24012,
-  }
+Truffle Dashboard can be used with Truffle or Hardhat. Please select a framework below for installation and configuration instructions.
 
-  networks: {
-    // ... network configurations, including the network named 'dashboard'
-  }
-}
-```
+- [Truffle](#usage-with-truffle)
+- [Hardhat](#usage-with-hardhat)
 
+## Usage with Truffle
 
-## Connecting to the dashboard
+### Connection
 
 To make connecting to the Truffle Dashboard easy, Truffle includes a built in network named "dashboard". This built in network automatically uses the port and host specified in the dashboard configuration or falls back to the default `http://localhost:24012`. This built in network can be used with all your deployments or scripts.
```

Comment:
Again, I know you didn't add this but "to _the_ Truffle Dashboard" sounds weird.